### PR TITLE
fix:#915 Unable to set initial time value for the time picker input

### PR DESCRIPTION
**Pull request by athulvraj**
### Description
There is no provision to set an inital time value(default value) to the Time Picker Component.
According to the documentation, value prop can be used to set initial values but its not working.
Fixed this issue in this.
fixes #issueid
Issue#915

### CODE REVIEW

**Comment by jbadan** [comment]
Try running `npm run test -- -u` to get your Travis build to pass

File: src/TimePicker/TimePicker.js [hunk 1]
```diff
@@ -75,6 +76,23 @@ class TimePicker extends Component {
...
+            time.second = timeArray[2].match(/\d+/)[0];
+            if (this.props.format12Hours) {
+                time.meridiem = timeArray[2].indexOf(this.props.localizedText.meridiemAM) !== -1 ? 0 : 1;
+            }
+        }
+        return time;
+    }
     // formatPlaceHolder = () => {
     //   let value = '';
     //   if (this.state.showHour) {
```
Comment (jbadan) [inline src/TimePicker/TimePicker.js]: Do you mind deleting this commented out code while you're working on this - it shouldn't be there. Thank you!

Comment (athulvraj) [reply src/TimePicker/TimePicker.js]: Its deleted

**Comment by athulvraj** [comment]
> Try running `npm run test -- -u` to get your Travis build to pass

Its passed locally but fails in Travis !!

**Comment by jbadan** [comment]
oh no! Is your branch up to date with master? I'll clear the cache in Travis too.

**Comment by athulvraj** [comment]
> oh no! Is your branch up to date with master? I'll clear the cache in Travis too.

Yes my branch has the latest code

File: src/TimePicker/__snapshots__/TimePicker.test.js.snap [hunk 2]
```diff
@@ -464,3 +464,49 @@ exports[`<TimePicker /> create time picker 10`] = `
...
+            value="10:30:34 pm"
+          />
+          <span
+            className="fd-input-group__addon fd-input-group__addon--button"
+          >
+            <button
+              aria-controls="rthHR811"
+              aria-expanded="false"
+              aria-haspopup="true"
+              className="fd-button fd-button--light sap-icon--fob-watch fd-input-group__button"
```
Comment (jbadan) [inline src/TimePicker/__snapshots__/TimePicker.test.js.snap]: This is why travis is failing - this should be `fd-button--transparent`

**Comment by athulvraj** [comment]
Done. All checks have passed !!

**Pull request merged**